Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -419,8 +415,7 @@ void Writer::WriteNestedMessage(
   if (stream_.bytes_written + size > stream_.max_size) {
     const char* errmsg =
         "Insufficient space in the output stream to write the given message";
-    FIREBASE_DEV_ASSERT_MESSAGE(false, errmsg);
-    status_ = Status(FirestoreErrorCode::Internal, errmsg);
+    FIREBASE_ASSERT_MESSAGE(false, errmsg);
```

Comment:
nit: `errmsg` text could be put inline directly in the assertion now that it isn't used twice.

Reply:
Done.